pulumi/backuprunner/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:18.12.1-alpine
```

Comment:
To align with all of our other dockerfiles, we can use the base image
```
ARG BASE_IMAGE

FROM $BASE_IMAGE
```

this will ensure we are using the same node/ts dependencies throughout the project

Reply:
This will require a bit of rework related to `node_modules` and workspace setup, so we can do a follow up